libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Microsoft.AdaptiveDialog.schema

```diff
@@ -22,6 +22,14 @@
             "description": "Value that will be passed back to the parent dialog.",
             "default": "dialog.result"
         },
+        "dialogs": {
+            "type": "array",
+            "title": "Referenced dialogs.",
+            "items": {
+                        "type": "string"
```

Comment:
> "type": "string" [](start = 23, length = 17)

This should be array of
        "dialog": {
            "oneOf": [
                {
                    "$kind": "Microsoft.IDialog",
                    "pattern": "^(?!(=)).*",
                    "title": "Dialog"
                },
                {
                    "$ref": "schema:#/definitions/equalsExpression",
                    "examples": [
                        "=settings.dialogId"
                    ]
                }
            ],
            "title": "Dialog name",
            "description": "Name of the dialog to call."
        },

Reply:
changed